implementation/src/main/java/io/smallrye/restclient/RestClientBuilderImpl.java

```diff
@@ -160,20 +156,17 @@ public RestClientBuilder executorService(ExecutorService executor) {
         resteasyClientBuilder.executorService(executor);
         resteasyClientBuilder.register(DEFAULT_MEDIA_TYPE_FILTER);
 
-        client = resteasyClientBuilder
-                .build();
+        client = resteasyClientBuilder.build();
 
-        actualClient = client.target(baseURI)
-                .proxyBuilder(aClass)
-                .classloader(classLoader)
-                .defaultConsumes(MediaType.APPLICATION_OCTET_STREAM)
+        actualClient = client.target(baseURI).proxyBuilder(aClass).classloader(classLoader).defaultConsumes(MediaType.APPLICATION_OCTET_STREAM)
```

Comment:
formatting :(

Reply:
Ah, I'm sorry. I accidentally formatted the whole file with WildFly defaults. I'll revert some of the changes ;-)